src/datatype/float.cpp

```diff
@@ -32,6 +34,17 @@ namespace datatype {
 		    Datatype(std::move(handle))
   {}
 
+  Float::Float(const Datatype &datatype):
+      Datatype(datatype)
+  {
+    if((datatype.get_class() != Class::FLOAT))
+    {
+      std::stringstream ss;
```

Comment:
Why do you do it with stream? Why not just throw std::runtime_exception("blabla") ?

Reply:
Good point. More or less a matter of habit. Stream makes it easier to create more elaborate error messages. Sometimes I use it sometimes not. I guess here I had something more elaborate in mind. Is this a real issue?

Reply:
Not REALLY an issue :) I just noticed it because it was not consistent between the two pull requests.